packages/slice-machine/src/features/onboarding/OnboardingProgressStepper.tsx

```diff
@@ -71,10 +74,14 @@ export const OnboardingProgressStepper = () => {
                   onSelect={() => showStep(step)}
                   description={step.description}
                   completed={isCompleted}
+                  readOnly={step.readonly}
                   endAdornment={
-                    <Icon name="chevronRight" size="small" color="grey11" />
+                    step.readonly !== true && (
+                      <Icon name="chevronRight" size="small" color="grey11" />
+                    )
                   }
                 >
+                  {isOnboardingCardVisibilityExperiment && index + 1 + " "}
                   {step.title}
```

Comment:
⚠️ let's use a string template here to make things clearer

Reply:
done

Reply:
Should we include the title as well?

Reply:
ok, changed again